quipucords/tests/api/credential/test_credential.py

```diff
@@ -1122,7 +1122,13 @@ def test_masked_data_serialization_list(self, django_client):
         """Test if data is masked as expected for list method."""
         results = []
         for input_data, output, _ in self.INPUT_OUTPUT_ID:
-            CredentialFactory(**input_data)
+            credential = CredentialFactory(**input_data)
+            output["created_at"] = credential.created_at.isoformat()
+            output["updated_at"] = (
+                credential.updated_at.isoformat()
+                if hasattr(credential, "updated_at")
```

Comment:
When could `credential.updated_at` not exist as an attribute? 

Is this just a weird side effect of using the factory? I think "real" credentials should always have this value.

Reply:
I guess u are correct. This field will be automatically updated whenever a change is made to the model instance. I will correct this.